### ISSUE TITLE
Option to change just the tab list?

### ISSUE THREAD

**Issue by SnakePin** [leadweedy/Firefox-Proton-Square#6]
Can an option to just change the design of the tabs and keep everything else as is, be added?

**Comment by leadweedy** [comment]
I hesitate to add this option, as it requires a lot of work for something that doesn't fit with the goal of the project of squaring the UI. I'd recommend checking out ![Lepton's Proton style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style) as it does pretty much that, only squaring the tabs. 

Otherwise, you can make your own `userChrome.css` by pulling lines 105-195 from this project:
```CSS
/* TAB BAR ------------------------------------------------------------------------------------------------- */

/* sets the height of the tab toolbar
otherwise will change height when tabs overflow [FIXED] */
/* tab height when playing audio issue fixed in FF 96.0.2+ */
/* https://bugzilla.mozilla.org/show_bug.cgi?id=1714276 */
/* height change when tab overflow fixed in FF 97 beta+ */
#TabsToolbar {height: var(--tab-bar-height, 35px) !important;}
:root{
  /* default value for tab-min-height = 36px */
  --tab-min-height: var(--tab-bar-height, 35px) !important;
}


/* Reduce left spacer on tab bar */
.titlebar-spacer[type="pre-tabs"] {
  display: none !important;
}


/* Tab - Connect to window */
.tab-background {
  border-radius: 0 !important;
  margin-top: 0 !important;
  margin-bottom: 0 !important;
}
.tab-content {
  margin-top: 0 !important;
  margin-bottom: 0 !important;
}
.tab-stack {
  margin-top: 0 !important;
  margin-bottom: 0 !important;
}


/* adds the little colored strip on top of the selected tab */
.tab-background:is([selected],[multiselected]) {
  box-shadow: inset 0 2px var(--custom-accent-color, #0a84ff) !important;
  border: 0 !important;
}


/* make context line (from container tabs) visible */
.tabbrowser-tab[selected] .tab-context-line {
  margin-top: 0 !important;
  margin-left: 0 !important;
  margin-right: 0 !important;
  border-radius: 0 !important;
}


/* realign the selected tab so it doesnt droop down by 2px */
.tabbrowser-tab[visuallyselected="true"]{
  z-index: 0 !important;
}


/* neighbouring tabs should "pinch" together */
.tabbrowser-tab {padding-inline: 0 !important;}


/* Unselected Tab - Divide line */
.tab-content::before,
.tab-content::after {
  content: "" !important;
  display: block !important;
  position: absolute !important;
  background-color: var(--custom-tab-divider-color, #535354) !important;
  width: 1px !important;
  height: 40px !important;
  transform: translateY(-20px) !important;
  opacity: 0 !important;
  transition: opacity 1s var(--ease-basic) !important;
}
.tab-content::before {left: 0 !important;}
.tab-content::after {right: 0 !important;}

.tabbrowser-tab[visuallyselected][style*=transform] + .tabbrowser-tab[style*=transform] .tab-content::after,
.tabbrowser-tab[style*=transform]:not([visuallyselected]) .tab-content::before,
.tabbrowser-tab[style*=transform] + .tabbrowser-tab:not([visuallyselected]) .tab-content::before,
.tabbrowser-tab:not([visuallyselected]):not(:hover):not([multiselected]) + .tabbrowser-tab:not([visuallyselected]):not(:hover):not([multiselected]) .tab-content::before,
#tabbrowser-tabs[hasadjacentnewtabbutton]:not([overflow]) .tabbrowser-tab[last-visible-tab]:not([visuallyselected]):not(:hover):not([multiselected]) .tab-content::after {
  opacity: var(--tab-separator-opacity) !important;
}


/* squares the mute button on hover */
.tab-icon-overlay:not([pinned], [sharing], [crashed]):is([soundplaying], [muted], [activemedia-blocked]) {
  border-radius: 0 !important;
}
```

**Comment by SnakePin** [comment]
Alright, thank you so much!
